### PR TITLE
2.6.4 post release

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Post release version bump to 2.6.5 and two small fixes in `AUTHORS` and `Makefile`.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,18 @@
+---------------------------
+udisks 2.6.5 (unreleased)
+---------------------------
+
+The storaged project provides a daemon, tools and libraries to access
```
Comment (vpodzime) [inline NEWS]: *udisks*

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+and manipulate disks, storage devices and technologies.
+
+Changes since 2.6.4:
```
Comment (vpodzime) [inline NEWS]: *2.6.5* I guess?

File: NEWS (hunk 1 above)
Comment (vojtechtrefny) [inline NEWS]: Fixed

File: NEWS (hunk 2 above)
Comment (vojtechtrefny) [inline NEWS]: Next version will be 2.6.5 so the changes will be _since 2.6.4_

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Pull request merged**